Samples/Example_ImGui_Docking/Example_ImGui_Docking.cpp

```diff
@@ -1237,8 +1237,8 @@ void Example_ImGuiRenderer::Update(float dt)
 
 		//PE: stop flickering when scrollbar goes on/off.
 		if (ImGui::GetCurrentWindow()->ScrollbarSizes.x > 0) {
-			ImGui::Text("");
-			ImGui::Text("");
+			ImGui::Text("%s", "");
```

Comment:
personally I would just go with `-Wno-format-zero-length` as I don't see the point, but this also works.